components/definitions/rewards.d.ts

```diff
@@ -253,14 +253,14 @@ declare namespace Rewards {
     args: Record<string, string>
   }
 
-  export interface AdsHistoryData {
+  export interface AdsHistory {
```

Comment:
nit: it's really hard to see a difference between `AdsHistory` and `AdHistory`, so maybe revert to `AdHistoryDetail`

Reply:
`AdsHistory` is the plural version of `AdHistory` due to needed serialization and deserialization. Prefer to leave the same as they are related